### ISSUE TITLE
jetbrains-toolbox gets reinstalled

### ISSUE THREAD

**Issue by BuZZ-dEE** [Antergos/antergos-packages#380]
Please see topic https://forum.antergos.com/topic/9889/jetbrains-toolbox-gets-reinstalled/

**Comment by lots0logs** [comment]
If you try to install one of the jetbrains apps individually from the AUR then you need to specifically use the AUR instead of using the repos. Packages providing what you are installing located in normal repos always take priority over AUR packages. So for example, if you are using yaourt to install phpstorm from the AUR, you should do it like this:

```sh
yaourt -S aur/phpstorm
```

**Comment by schmunk42** [comment]
I get this error message since an update today

```
$ jetbrains-toolbox 
Unknown command-line options: ("disable-seccomp-filter-sandbox")
```

**Comment by andrebrait** [comment]
@schmunk42 I fixed it in #381
